### PR TITLE
doc debug_dumpAddress

### LINKED ISSUE #40: Doc debug_dumpAddress
PR: https://github.com/ConsenSys/quorum/pull/1072

### PR DESCRIPTION
**Pull request by alexandratran**
## Fixed Issue(s)

Document debug_dumpAddress. Add page for debug namespace.

fixes #40

### CODE REVIEW

File: docs/Reference/APIs/DebugAPIs.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## APIs
+
+### `debug_dumpAddress`
+
+Retrieves the state of an address at a given block.
+
+#### Parameters
+
+* `address`: Account address to retrieve the state of.
```
Comment (bgravenorst) [inline docs/Reference/APIs/DebugAPIs.md:15]: Nit: "Account address of the state to retrieve.

File: docs/Reference/APIs/DebugAPIs.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* `blockNr`: Integer representing a block number or one of the string tags `latest` (the last block mined) or `pending`
+  (the last block mined plus pending transactions).
+
+#### Returns
+
+* state of the account address
+
+!!! Example
+
+    === "curl HTTP"
```
Comment (bgravenorst) [inline docs/Reference/APIs/DebugAPIs.md:26]: "JSON RPC"?

**Review by bgravenorst — COMMENTED** [review]
Minor comments

File: docs/Reference/APIs/DebugAPIs.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+### `debug_dumpAddress`
+
+Retrieves the state of an address at a given block.
+
+#### Parameters
+
+* `address`: Account address of the state to retrieve.
+
+* `blockNr`: Integer representing a block number or one of the string tags `latest` (the last block mined) or `pending`
```
Comment (macfarla) [inline docs/Reference/APIs/DebugAPIs.md:17]: blockNumber

**Review by nmvalera — APPROVED** [review]
LGTM

**Pull request merged**